dask/dataframe/tests/test_groupby.py

```diff
@@ -1207,6 +1209,27 @@ def test_shuffle_aggregate_defaults(shuffle_method):
     assert any("shuffle" in l for l in dsk.layers)
 
 
+@pytest.mark.parametrize("spec", [{"c": "median"}, {"b": np.median, "c": np.max}])
+@pytest.mark.parametrize("keys", ["a", ["a", "d"]])
+def test_aggregate_median(spec, keys, shuffle_method):
```

Comment:
Do we have test coverage for calling `.median(...)` instead of passing a spec (with median) to `aggregate`?

Reply:
Yes, that's covered by including it in `AGG_FUNCS` above, so it is exercised in a number of places.